### PR TITLE
TypeScript conversion batch 1

### PR DESCRIPTION
**Pull request by drosenberg123**
- converted `AccessibilityObject` and `Roles` to TS
- split `Roles` helpers into new `utils`
- (minor) updated test-app/tests to match changes
- note my jetbrains ide complains about 3 args for displayObject.dispatchEvent() even with @ts-ignore but it lints OK

### CODE REVIEW

File: src/RoleObjects/AccessibilityObject.ts [hunk 1]
```diff
@@ -15,15 +65,19 @@ export default class AccessibilityObject {
...
      */
     this._reactProps = {
       key: this._displayObject.id,
       id: this.domId,
-      onFocus: (evt) => {
+      onFocus: (evt: Event) => {
         const cancelled = this._displayObject.dispatchEvent(
           'focus',
           false,
+          // @ts-ignore
```
Comment (drosenberg123) [inline src/RoleObjects/AccessibilityObject.ts:80]: - there is a mismatch in the number of args in `@types/createjs` (https://github.com/DefinitelyTyped/DefinitelyTyped) so we had to override here
- we intend on opening a PR to resolve

File: src/RoleObjects/AccessibilityObject.ts [hunk 2]
```diff
@@ -32,10 +86,11 @@ export default class AccessibilityObject {
           evt.preventDefault();
         }
       },
-      onBlur: (evt) => {
+      onBlur: (evt: Event) => {
         const cancelled = this._displayObject.dispatchEvent(
           'blur',
           false,
+          // @ts-ignore
```
Comment (drosenberg123) [inline src/RoleObjects/AccessibilityObject.ts:93]: - same as above

**Review by mmoss — APPROVED** [review]
👍

**Review by tussharrane — APPROVED** [review]
👍🏽

**Pull request merged**